ui/src/app/job-list/table/table.component.html

```diff
@@ -3,12 +3,18 @@
   <!-- Job name column -->
   <ng-container matColumnDef="Job">
     <mat-header-cell *matHeaderCellDef>
-      <mat-checkbox class = "checkbox" [checked]="allSelected" (change)="toggleSelectAll()">
+      <mat-checkbox class = "checkbox"
+                    (change)="$event ? toggleSelectAll() : null"
```

Comment:
Style guide reference: https://angular.io/guide/styleguide#put-presentation-logic-in-the-component-class

I'll leave it to your judgment, it's a pretty simple operation so I don't have an overwhelming preference either way.

Reply:
Thanks for the link. I don't think it should be a hard and fast rule, but I've applied it in the checked/indeterminate cases, so that I can at least comment the logic there.